Dockerfile.testing

```diff
@@ -1,24 +1,21 @@
 # For building gcr.io/forgerock-io/secret-agent-testing:latest
 
-FROM openjdk:11-jdk-slim
+FROM openjdk:11-jre-slim
 ENV CGO_ENABLED=0 GOOS=linux GOARCH=amd64
-CMD ["bash"]
-RUN apt-get update && apt-get install -y curl git-core
-RUN curl -O https://dl.google.com/go/go1.13.10.linux-amd64.tar.gz && \
-    SUM=$(sha256sum go1.13.10.linux-amd64.tar.gz | awk '{print $1}') && \
+RUN apt-get update && apt-get install -y curl git-core make && apt-get clean all
+RUN curl -LO https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz && \
+    SUM=$(sha256sum go1.14.2.linux-amd64.tar.gz | awk '{print $1}') && \
     if [ "$SUM" != "8a4cbc9f2b95d114c38f6cbe94a45372d48c604b707db2057c787398dfbf8e7f"]; then echo "Failed checksum"; exit 1; fi && \
-    tar xvf go1.13.10.linux-amd64.tar.gz && \
+    tar xf go1.14.2.linux-amd64.tar.gz && \
     chown -R root:root ./go && \
     mv go /usr/local && \
-    rm go1.13.10.linux-amd64.tar.gz
-ENV PATH="/usr/local/go/bin:${PATH}" GOPATH="/root/go"
-WORKDIR /root/go/src/github.com/ForgeRock/secret-agent
-RUN addgroup --gid 11111 forgerock && \
-    adduser --shell /bin/bash --home /home/forgerock --uid 11111 --disabled-password --ingroup root --gecos forgerock forgerock && \
-    chown -R forgerock:root /home/forgerock
+    rm go1.14.2.linux-amd64.tar.gz
+
+RUN curl -L https://go.kubebuilder.io/dl/2.3.1/${GOOS}/${GOARCH} | tar -xz -C /tmp/ && \
```

Comment:
Are GOOS and GOARCH definitely available as regular env vars? Depending on what they can at the base URL, I could see this possibly failing silently if those are not regular env vars.

Reply:
we set them ourselves in the dockerfile 😁 
https://github.com/ForgeRock/secret-agent/blob/3df51a69f186c0138c71c8ca1196b18878aa6b73/Dockerfile.testing#L4

Reply:
Doh. Good call. Hah.